src/main/java/com/metsci/laproc/uicomponents/GraphDisplayer.java

```diff
@@ -20,6 +23,19 @@
     private IAction<GraphPoint[]>[] pointClickActions;
     private Graph graph;
 
+    static final float[][] possibleColors = new float[8][4];
+
+    static {
+        possibleColors[0] = GlimpseColor.fromColorRgb(0f, 0f, 0f);
+        possibleColors[1] = GlimpseColor.fromColorRgb(1f, 0f, 0f);
+        possibleColors[2] = GlimpseColor.fromColorRgb(0f, 1f, 0f);
+        possibleColors[3] = GlimpseColor.fromColorRgb(0f, 0f, 1f);
+        possibleColors[4] = GlimpseColor.fromColorRgb(0.2f, 0.5f, 0.5f);
+        possibleColors[5] = GlimpseColor.fromColorRgb(0.4f, 0.4f, 0f);
+        possibleColors[6] = GlimpseColor.fromColorRgb(1f, 0f, 1f);
+        possibleColors[7] = GlimpseColor.fromColorRgb(0.4f, 0f, 0.4f);
```

Comment:
Where are these changes showing up in here? Did my branch get merged into this one?